pennylane/compiler/qjit_api.py

```diff
@@ -696,7 +706,8 @@ def _call_capture_enabled(self, *init_state):
             *jaxpr_body_fn.consts,
             *flat_args,
             jaxpr_body_fn=jaxpr_body_fn.jaxpr,
-            n_consts=len(jaxpr_body_fn.consts),
+            consts_slice=consts_slice,
+            args_slice=args_slice,
         )
```

Comment:
On exactly the parts that you did not change here: It would of course be nice to have `start` and `stop` in everywhere to get consistency both internally and with standard slice/iteration syntax.
I suppose this would cause a breaking change?

Reply:
standardizing to `start, stop, step`. I think `ForLoopCallable` is internal enough we can rename things without issue.